mappings/net/minecraft/client/MinecraftClient.mapping

```diff
@@ -35,6 +35,7 @@ CLASS cpk net/minecraft/client/MinecraftClient
 	FIELD aN fpsCounter I
 	FIELD aO tutorialManager Ldrd;
 	FIELD aP isWindowFocused Z
+	FIELD aQ profilerSectionCurrentlyOpen Ljava/lang/String;
```

Comment:
It refers to being the section that's currently open in the Shift-F3 pie chart. But I agree that `profilerSectionCurrentlyOpen` is too long, so I changed it to `openProfilerSection` since that's more descriptive than "active". Is that ok?